receiver/splunkenterprisereceiver/metadata.yaml

```diff
@@ -6,4 +6,99 @@ status:
+  bucket.dir:
+    description: The bucket super-directory (home, cold, thawed) for each index
+    type: string
+
+metrics:
+  splunk.license.index.usage:
+    enabled: true
+    description: Gauge tracking the indexed license usage per index
+    unit: GBy
+    gauge:
+      value_type: double
+    attributes: [index.name]
+  # 'services/server/introspection/indexer'
+  splunk.server.introspection.indexer.throughput:
+    enabled: true
+    description: Gauge tracking average KBps throughput of indexer
+    unit: KBy
+    gauge:
+      value_type: double
+    # attribute `status` can be one of the following `normal`, `throttled`, `stopped`
+    attributes: [status]
+  # 'services/data/indexes-extended'
+  splunk.data.indexes.extended.bucket.count:
+    enabled: true
+    description: Count of buckets per index
+    unit: "{Buckets}" 
+    gauge:
+      value_type: int
+    attributes: [index.name]
+  splunk.data.indexes.extended.event.count:
+    enabled: true
+    description: Count of events for index, excluding frozen events. Approximately equal to the event_count sum of all buckets.
+    unit: "{Events}" 
+    gauge:
+      value_type: int
+    attributes: [index.name]
+  splunk.data.indexes.extended.raw.size:
+    enabled: true
+    description: Cumulative size (fractional MB) on disk of the <bucket>/rawdata/ directories of all buckets in this index, excluding frozen
+    unit: MBy
+    gauge:
+      value_type: double
+    attributes: [index.name]
+  splunk.data.indexes.extended.total.size:
+    enabled: true
+    description: 	Size (fractional MB) on disk of this index
+    unit: MBy
+    gauge:
+      value_type: double
+    attributes: [index.name]
+  ## Worth breaking down `bucket_dirs`? Counts of hot/cold/warm buckets, sizes of individual buckets, etc?
+  ## Examples set to `enabled: false` by default
+  splunk.data.indexes.extended.bucket.dirs.size:
+    enabled: false
+    description: 	Size (fractional MB) on disk of this bucket super-directory
+    unit: MBy
+    gauge:
+      value_type: double
+    attributes: [index.name, bucket.dir]
+  splunk.data.indexes.extended.bucket.dirs.count:
+    enabled: false
+    description: Count of buckets
+    unit: "{Buckets}" 
+    gauge:
+      value_type: int
+    attributes: [index.name, bucket.dir]
+  splunk.data.indexes.extended.bucket.event.count:
+    enabled: false
+    description: Count of events in this bucket super-directory
+    unit: "{Events}" 
+    gauge:
+      value_type: int
+    attributes: [index.name, bucket.dir]
+  splunk.data.indexes.extended.bucket.hot.count:
+    enabled: false
+    description: (If size > 0) Number of hot buckets
+    unit: "{Buckets}" 
+    gauge:
+      value_type: int
+    attributes: [index.name, bucket.dir]
+  splunk.data.indexes.extended.bucket.warm.count:
+    enabled: false
+    description: (If size > 0) Number of warm buckets
+    unit: "{Buckets}" 
+    gauge:
+      value_type: int
+    attributes: [index.name, bucket.dir]  
```

Comment:
Can you please keep the implemented metrics only here? Let's add others along with the implementation.

Reply:
oh, yea absolutely